packages/api/tests/es/test-stats.js

```diff
@@ -0,0 +1,132 @@
+});
+
+test.afterEach(async (t) => {
+  await esClient.indices.delete({ index: t.context.esIndex });
+});
+
+test.after.always(async () => {
+  await collectionModel.deleteTable();
+  await granuleModel.deleteTable();
+  await s3.recursivelyDeleteS3Bucket(process.env.system_bucket);
+});
+
+test.serial('Stats returns one collection when a collecion is indexed', async (t) => {
+  const collection = fakeCollectionFactory();
+  await indexer.indexCollection(esClient, collection, t.context.esAlias);
+
+  const stats = new Stats({}, null, process.env.ES_INDEX);
+  const queryResult = await stats.query();
+
+  t.is(queryResult.collections.value, 1);
+});
+
+test.serial('Stats returns one granule when a granule is indexed', async (t) => {
+  const granule = fakeGranuleFactoryV2();
+  await indexer.indexGranule(esClient, granule, t.context.esAlias);
+
+  const stats = new Stats({}, null, process.env.ES_INDEX);
+  const queryResult = await stats.query();
+
+  t.is(queryResult.granules.value, 1);
+});
+
+test.serial('Stats returns correct granule errors', async (t) => {
+  await Promise.all([
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2(), t.context.esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2(), t.context.esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2({ status: 'failed' }), t.context.esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2({ status: 'failed' }), t.context.esAlias)
+  ]);
+
+  const stats = new Stats({}, null, process.env.ES_INDEX);
+  const queryResult = await stats.query();
+
+  t.is(queryResult.granules.value, 4);
+  t.is(queryResult.errors.value, 2);
+});
+
+test.serial('Count returns 0 if there are no granules', async (t) => {
+  const stats = new Stats({}, null, process.env.ES_INDEX);
+  const countResult = await stats.count();
+
+  t.is(countResult.meta.count, 0);
+});
+
+test.serial('Count returns correct granule and collection count', async (t) => {
+  await Promise.all([
+    indexer.indexCollection(esClient, fakeCollectionFactory(), t.context.esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2(), t.context.esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2(), t.context.esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2({ status: 'failed' }), t.context.esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2({ status: 'failed' }), t.context.esAlias)
+  ]);
+
+  const stats = new Stats({}, 'granule', process.env.ES_INDEX);
+  const countResult = await stats.count();
+
+  t.is(countResult.meta.count, 4);
+  t.deepEqual(countResult.count, [
+    { key: 'completed', count: 2 },
+    { key: 'failed', count: 2 }
+  ]);
+
+  const collectionStats = new Stats({}, 'collection', process.env.ES_INDEX);
+  const collectionCountResult = await collectionStats.count();
+
+  t.is(collectionCountResult.meta.count, 1);
+
+  const req = { query: { type: 'granule '} };
+  console.log(get(req, 'query.type', null));
```

Comment:
should this be a test here?

Reply:
No I moved those specific tests to that other file. I mean yes I am still sloppy, but that was quick testing my fix before I deployed, then I accidentally left it in. I added all of the tests in the other file to test what I had been testing here.